### ISSUE TITLE
Exception raised when a model is set on the table  after attaching the filter header

### ISSUE THREAD

**Issue by coderazzi** [coderazzi/tablefilter-swing#7]
(Original issue 7 created by coderazzi on 2011-03-05T14:11:44+00:00)

```
1- Create the JTable
2- Create the TableFilterHeader and attach it to the table
3- Set a model on the table

The exception happens when setting the comparator on the underlying row sorter: 

Exception in thread &quot;main&quot; java.lang.IndexOutOfBoundsException: column beyond
range of TableModel
	at javax.swing.DefaultRowSorter.checkColumn(DefaultRowSorter.java:1202)
	at javax.swing.DefaultRowSorter.setComparator(DefaultRowSorter.java:744)
	at net.coderazzi.filters.gui.editor.FilterEditor.initComparator(FilterEditor.java:337)
	at net.coderazzi.filters.gui.editor.FilterEditor.&lt;init&gt;(FilterEditor.java:137)
```

Original issue reported on code.google.com by `coderazzi` on 2011-03-05 14:11:44

**Comment by coderazzi** [comment]
(Original issue 7 last updated on 2015-04-26T00:26:12.773020+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by coderazzi** [comment]
(Original comment by coderazzi on 2011-03-14T03:37:21+00:00)

```
Solved on version 4.1.0
```

Original issue reported on code.google.com by `coderazzi` on 2011-03-14 03:37:21

**Comment by coderazzi** [comment]
(Original comment by coderazzi on 2015-04-26T00:26:12.785355+00:00)

Already resolved (previously, before migrating to BitBucket)
